jekyll/_cci2/faq.md

```diff
@@ -283,6 +283,19 @@ You can choose to pay for premium features per active user, compute, and optiona
 - Docker Layer Caching (DLC) is paid for with credits per usage, similar to
   compute credits.
 
+#### What constitutes an _Active User_?
+{:.no_toc}
+
+An `active user` is any user who meets the following criteria:
+
+- Anyone triggering the use of compute resources on non-OSS projects.
```

Comment:
I was probably unclear in my slack post, so suggested tweaks to clarify

> An `active user` is any user who triggers the use of compute resources on non-OSS projects. This includes activities such as:
- Commits from users that trigger builds, including PR Merge commits.
- Re-running jobs in the CircleCI web application, including SSH debug.
- Approving manual jobs (approver will be considered the actor of all downstream jobs).


and keep the footer note about OSS 👍

Reply:
@teesloane - do we usually include links to other docs from FAQ?  If so than the [SSH Debug](https://circleci.com/docs/2.0/ssh-access-jobs/), [Manual approval jobs](https://circleci.com/docs/2.0/workflows/#holding-a-workflow-for-a-manual-approval), and [OSS](https://circleci.com/docs/2.0/oss/) might be good candidates.

Reply:
updated! Thanks @eddiewebb !